test/microphysics1M_tests.jl

```diff
@@ -52,6 +52,22 @@ function test_microphysics1M(FT)
         end
     end
 
+    TT.@testset "1M_microphysics RadarReflectivity" begin
+
+        # some example values
+        ρ_air = FT(1.0066405993926746)
+        q_rai = FT(0.00018261331850998665)
+
+        TT.@test CM1.radar_reflectivity(rain, q_rai, ρ_air) ≈
+                 FT(12.165348140278356) atol = 0.2
```

Comment:
If the absolute toleranace is `0.2` then we don't really need all those numbers after the decimal point, right?

Also, could we make the inputs be simply
```
ρ_air = FT(1)
q_rai = FT(0.18e-3)
```